CHANGELOG_SDK.md

```diff
@@ -1,11 +1,17 @@
 # Changelog SDK
 
-## 2023-06-20 `v3.1.0`
+## 2023-06-22 `v3.1.0`
```

Comment:
the idea is to omit the concrete day and just mention year and month. IMHO the day is pretty useless and is not practical to guess the day of a future merge.

Reply:
ok, then there will be many same months now, but ok. I am working on the test files now...

Reply:
will adapt the cahngelog directly on main